source/includes/_inspections.md

```diff
@@ -1,6 +1,5 @@
 # Inspections
 
-## Deprecated
```

Comment:
This and similar fixes are needed also because this is not a unique header, which causes the navigation problems. I decided to remove instead of renaming because it felt to me that the deprecation notice below is enough.